src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -154,6 +159,22 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 				this._undoStack.maxStackSize = newStackSize;
 				this._redoStack.maxStackSize = newStackSize;
 			}
+			if (e.affectsConfiguration('notebook.renderTablesInHtml')) {
+				this._htmlMarkdownConverter = this._instantiationService.createInstance(HTMLMarkdownConverter, this.notebookUri);
+				this.updateCellSource();
+				if (this.isEditMode) {
+					this._notificationService.prompt(Severity.Info, promptReloadNotebooks, [{
+						label: 'Reload',
+						run: () => {
+							this.markdowncodeCell.map(code => code.refreshCell());
```

Comment:
forEach - map should generally just be used when you want to convert stuff into another form